package.json

```diff
@@ -1,8 +1,8 @@
 {
   "name": "ship-components-ago",
-  "version": "0.1.0",
+  "version": "0.1.1",
```

Comment:
According to semver since we're adding a new feature, localization, this would be a minor update and therefore we need to bump the version to 0.2.0